xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -813,9 +814,44 @@ public static string PemEncodeSigningRequest(CertificateRequest request, PkcsSig
         <param name="signatureGenerator">The signature generator with which to sign the request.</param>
         <summary>Creates an ASN.1 DER-encoded PKCS#10 CertificationRequest representing the current state of the current object using the provided signature generator.</summary>
         <returns>A DER-encoded certificate signing request.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+When submitting a certificate signing request via a web browser, or other graphical or textual
+              interface, the input is frequently expected to be in the PEM (Privacy Enhanced Mail) format,
+              instead of the DER binary format.
+
+          ]]></format>
+        </remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="signatureGenerator" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.InvalidOperationException">
+          <see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains a <see langword="null" /> value.
+
+-or-
+
+<see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains an entry with a <see langword="null" /><see cref="P:System.Security.Cryptography.AsnEncodedData.Oid" /> value.
+
+-or-
+
+<see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains an entry representing the PKCS#9
```

Comment:
This change seems to have caught docs that were updated, but not docs for new members.  That's... fine... except this `<see>` is linking against a new member.  I'm mainly confused because the `seealso` that got added in this same PR is not present:  https://github.com/dotnet/runtime/blob/e71a9583b4d6c9bd97edd87cda7f98f232f63530/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRequest.cs#L332

Reply:
Might be a bug in the tool. It's supposed to be able to port standalone `<see>` elements (outside of summaries, returns, remarks or exceptions). Maybe the tool is not able to port `<seealso>` items, and if it does, then it definitely has a bug.

I can manually add it in a separate suggestion comment.

Reply:
> It's supposed to be able to port standalone <see> elements (outside of summaries, returns, remarks or exceptions)

Hm.  My understanding of xmldoc is `<see>` is only valid as a child of one of those things, and at the "top" level only `<seealso>` is valid.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#see

> The `<see>` tag lets you specify a link from within text. Use `<seealso>` to indicate that text should be placed in a See Also section.

Reply:
Oh I see what the problem is: That API was added [just 6 days ago](https://github.com/dotnet/runtime/blame/22ba36d5b0e78c984d8e0b24ae389cabd345ec30/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRequest.cs#L464), so the tool didn't find it.

The documentation for those new APIs will get ported after I send the latest ref assemblies (Prev7) to Genevieve.

Since Prev7 was released today, I can start the process tomorrow.